docs/guides/instances.mdx

```diff
@@ -157,8 +155,8 @@ sampler2 = SamplerV2(mode=backend2)
 If you do not specify an instance, the code will select one in the following order:
 
 1. If your account only has access to one instance, it is selected by default.
-2. If your account has access to multiple instances but only one can access the requested QPU, the instance with access is selected.
-3. In all other cases, the code selects the first instance other than the Open Plan instance that has access to the quantum computer.
+2. If your account has access to multiple instances but only one can access the requested QPU, the instance with access is selected. If this instance is not associated with a free plan, a cost might be incurred.
+3. In all other cases, the code selects the first instance other than the Open Plan instance that has access to the quantum computer. If this instance is not associated with a free plan, a cost might be incurred.
```

Comment:
Seems strange to me that a cost only "might" be incurred when an instance is selected that does not use a free plan. Wouldn't it definitely incur a cost?

Reply:
Well, I figured you wouldn't get charged if you used a simulator, or if you have free credits or something.  Too much hair splitting?

Reply:
Fair enough!